### ISSUE TITLE
Sender in TokenTranser message

### ISSUE THREAD

**Issue by 0xMaxyz** [howardpen9/jetton-implementation-in-tact#19]
Hi Peng,
I was testing your Jetton implementation and I came across this in the [messages](https://github.com/howardpen9/jetton-implementation-in-tact/blob/f1e820aa33bdf7765dcbb4c00db39e39409d382e/sources/messages.tact#L19)
```tact
message(0xf8a7ea5) TokenTransfer {
    query_id: Int as uint64;
    amount: Int as coins;
    sender: Address;
    response_destination: Address?;
    custom_payload: Cell?;
    forward_ton_amount: Int as coins;
    forward_payload: Slice as remaining; 
}
```
shouldn't the `sender` in `TokenTransfer` message be `to` or `destination`? this is ambiguous, first I thought I have to put my address there and send the message to wallet contract of the receiver, then I checked the `receive` code for `TokenTransfer` and found that this is actually the address of the receiver of tokens, am I wrong here?

**Comment by GemsGame** [comment]
the token standard is
transfer#0f8a7ea5 query_id:uint64 amount:(VarUInteger 16) destination:MsgAddress
                 response_destination:MsgAddress custom_payload:(Maybe ^Cell)
                 forward_ton_amount:(VarUInteger 16) forward_payload:(Either Cell ^Cell)
                 = InternalMsgBody;
                 
 where is sender?)

**Comment by 0xMaxyz** [comment]
> the token standard is transfer#0f8a7ea5 query_id:uint64 amount:(VarUInteger 16) destination:MsgAddress response_destination:MsgAddress custom_payload:(Maybe ^Cell) forward_ton_amount:(VarUInteger 16) forward_payload:(Either Cell ^Cell) = InternalMsgBody;
> 
> where is sender?)


Yeah, you're right about the standard, but my question was about the Tact implementation in this repo (token transfer message), pls check this [link](https://github.com/howardpen9/jetton-implementation-in-tact/blob/f1e820aa33bdf7765dcbb4c00db39e39409d382e/sources/messages.tact#L19)
